presto-orc/src/main/java/com/facebook/presto/orc/AbstractOrcRecordReader.java

```diff
@@ -190,49 +192,54 @@ public AbstractOrcRecordReader(
         // it is possible that old versions of orc use 0 to mean there are no row groups
         checkArgument(rowsInRowGroup > 0, "rowsInRowGroup must be greater than zero");
 
+        long fileRowCount = 0;
+
         // sort stripes by file position
-        List<StripeInfo> stripeInfos = new ArrayList<>();
+        IntArrayList sortedStripeIds = new IntArrayList(fileStripes.size());
```

Comment:
I wanted to avoid boxing integers, `Array.sort(int[])` doesn't support custom comparators - that's why I used IntArrayList. Will change it to `List<Integer>`.